packages/hofexpress_app/Classes/Controller/OrderItemsController.php

```diff
@@ -116,11 +117,11 @@ public function updateAction(\Hulk\Hofexpress\Domain\Model\OrderItems $orderItem
 
     /**
      * action delete
-     * 
+     *
      * @param \Hulk\HofexpressApp\Domain\Model\OrderItems $orderItems
      * @return void
      */
-    public function deleteAction(\Hulk\Hofexpress\Domain\Model\OrderItems $orderItems)
+    public function deleteAction(\Hulk\HofexpressApp\Domain\Model\OrderItems $orderItems)
```

Comment:
These adjustments in PHP namespace were required in order to overcome `ClassSchema` error messages during parsing the controller (which happens in order to determine parameters of these action methods).